packages/x-components/src/design-system/variables.scss

```diff
@@ -24,13 +24,13 @@ $sizes: (
   '09': 48px,
   '10': 56px,
   '11': 64px,
-  '12': 72px,
+  '12': 80px,
   '13': 80px,
```

Comment:
I think it should be 72px (each size increases 8px respect to the previous one)

Reply:
Yes it is, because some values doesn't exist anymore in new DS, so we take the nearest for old DS, even this implies repeated values.

This values should be not used soon.
For older setups, whenever we updated them, we can choose to keep in old DS or update completely to new one.